marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/mastering/test-data/content/matchSourceQuery.json

```diff
@@ -0,0 +1,14 @@
+{
+  "envelope": {
+    "headers": {
+      "sources": [{
+        "name": "matchingSourceJSON"
```

Comment:
Pinging @rahulvudutala  - these will need to be updated once the datahubSourceName field index is pointing at datahubSourceName instead of name.